### ISSUE TITLE
Kandinsky-2.2 yellow shades

### ISSUE THREAD

**Issue by samitkumarsinha** [ai-forever/Kandinsky-2#91]
Noticed a yellow tint on most 2.2 images. image looks like sketches and illustration. Image quality deteriorates, poorer than 2.1.

**Comment by Kate-Pod** [comment]
also noticed it. Pictures are often generated in an unpleasant yellow color, closer to acid

**Comment by joel-simon** [comment]
@samitkumarsinha Try using the empty image embed as the negative prompt instead of the embedding of the empty string. Changing this fixed it for me.

**Comment by samitkumarsinha** [comment]
Nope. No need. I have stopped using kandinsky since the arrival of SDXL.
If they fix the issue in kandinsky next version, I'll join again.
Thanks anyway
